File: totosuki/ABC/B/112.py
N, T = map(int, input().split())
min_cost = 1001

for _ in range(N):
  c, t = map(int, input().split())
  if t <= T:
    min_cost = min(min_cost, c)

print(min_cost) if min_cost != 1001 else print("TLE")

File: OCH_Tao/ABC/B/112.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const [N, T] = ((input.shift()).split(" ")).map(x => parseInt(x, 10));
let cnt = [];
for (const X of input) {
  const [c, t] = (X.split(" ")).map(x => parseInt(x, 10));
  if (t <= T) {
    cnt.push(c);
  }
}
if (cnt.length === 0) {
  console.log("TLE");
} else {
  console.log(Math.min(...cnt));
}